app/models/saved_claim/education_benefits/va_1995.rb

```diff
@@ -2,4 +2,46 @@
 
 class SavedClaim::EducationBenefits::VA1995 < SavedClaim::EducationBenefits
   add_form_and_validation('22-1995')
+
+  # pulled from https://github.com/department-of-veterans-affairs/vets-website/blob/f27b8a5ffe4e2f9357d6c501c9a6a73dacdad0e1/src/applications/edu-benefits/utils/helpers.jsx#L100
+  BENEFIT_TITLE_FOR_1995 = {
+    'chapter30' => 'Montgomery GI Bill (MGIB or Chapter 30) Education Assistance Program',
+    'chapter33' => 'Post-9/11 GI Bill (Chapter 33)',
+    'chapter1606' => 'Montgomery GI Bill Selected Reserve (MGIB-SR or Chapter 1606) Educational Assistance Program',
+    'chapter32' => 'Post-Vietnam Era Veterans’ Educational Assistance Program (VEAP or chapter 32)'
```

Comment:
What about the additional options that the form shows?

![Screen Shot 2023-05-01 at 1 48 57 PM](https://user-images.githubusercontent.com/4740018/235519260-517e8cac-5460-4308-8b07-d926b8594a8a.png)